ocaml/testsuite/tests/typing-layouts-float64/basics.ml

```diff
@@ -266,7 +287,10 @@ Error: Layout mismatch in final type declaration consistency check.
        clever enough to propagate layouts through variables in different
        declarations. It is also not clever enough to produce a good error
        message, so we'll say this instead:
-         'a has layout float64, which does not overlap with value.
+         The layout of 'a is float64, because
```

Comment:
Actually, that message is somewhat decent! The type inference isn't clever, but at least the error is. Could reword the message. But humility is a virtue, and so I don't actually think we need to change anything. But I'm pleased. (Resolve after reading.)